ql/src/test/org/apache/hadoop/hive/ql/exec/repl/TestAtlasDumpTask.java

```diff
@@ -96,4 +105,13 @@ public void testAtlasDumpMetrics() throws Exception {
     Assert.assertTrue(eventDetailsCaptor
         .getAllValues().get(1).toString().contains("{\"dbName\":\"srcDB\",\"dumpEndTime\""));
   }
+
+  @Test
+  public void testAtlasRestClientBuilder() throws SemanticException, IOException {
+    mockStatic(UserGroupInformation.class);
+    when(UserGroupInformation.getLoginUser()).thenReturn(mock(UserGroupInformation.class));
+    AtlasRestClientBuilder atlasRestCleintBuilder = new AtlasRestClientBuilder("http://localhost:31000");
+    AtlasRestClient atlasClient = atlasRestCleintBuilder.getClient(conf);
+    Assert.assertTrue(atlasClient != null);
```

Comment:
hive in test repl is set to true. It will return a No Op Client which will never be null.

Reply:
HiveConf is mocked, so hive in test is not present(so false).